nixos/modules/config/pulseaudio.nix

```diff
@@ -135,6 +177,7 @@ in {
 
     (mkIf cfg.enable {
       environment.systemPackages = [ cfg.package ];
+      environment.sessionVariables = moduleEnvVars;
```

Comment:
An idea for potential improvement: instead add to environment not raw `cfg.package` but a wrapped one with `makeWrapper`ed binaries adding those variables. Less pollution in the environment and more robustness.